app/src/androidTest/java/ch/hikemate/app/ui/map/MapScreenTest.kt

```diff
@@ -166,6 +171,64 @@ class MapScreenTest : TestCase() {
     composeTestRule.onNodeWithTag(MapScreen.TEST_TAG_EMPTY_HIKES_LIST_MESSAGE).assertIsDisplayed()
   }
 
+  @Test
+  fun hikesListDisplaysHikeNameIfAvailable() {
+    // Given
+    val hikeName = "My test hike whatever"
+    val hike = HikeRoute("1", Bounds(-1.0, -1.0, 1.0, 1.0), listOf(), hikeName, null)
+    `when`(hikesRepository.getRoutes(any(), any(), any())).thenAnswer {
+      val onSuccess = it.getArgument<(List<HikeRoute>) -> Unit>(1)
+      onSuccess(listOf(hike))
+    }
+
+    // When
+    setUpMap()
+    listOfHikeRoutesViewModel.setArea(BoundingBox(0.0, 0.0, 0.0, 0.0))
+    composeTestRule.waitUntil(timeoutMillis = 5000) {
```

Comment:
Question: Is there another way to do that ? This would really make the tests longer in the long term

Reply:
Discussing this issue in the above conversation, closing this one.